client/src/pages/OrganizationProfile.js

```diff
@@ -193,12 +193,12 @@ const OrganizationProfile = () => {
       const offerHelp = Object.values(objectives).some((val) => val === true);
       const { address } = location;
       const nameArr = name.split(" ");
-      if (nameArr.length < 2) {
+      if (nameArr.length < 1) {
```

Comment:
Is this working?
if `nameArr = [ "Firstname" ];` <- Length 1 false
It's only true with `nameArr = [];` and it's useless.

Reply:
no its not working! I was trying to use similar method to display 2 initials instead of 1

Reply:
and based of my comment below, you don't have to change anything in this line here.